tests/test_backends.py

```diff
@@ -1,8 +1,10 @@
 import pytest
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
 from django.test import override_settings
 from django.urls import reverse
+from django.utils.module_loading import import_string
 
-from tests import test_settings
```

Comment:
Why?

Reply:
We are using `django.conf.settings`,  hence, I don't see any need for it. Apart from it, this setting would always be static. But the settings fetched from `django.conf.settings`  would always be the updated one, corresponding to the overridden settings.

Reply:
Ah, so I visited the code again and found out that we're configuring the pytest with these settings.

Reply:
Yeah, there has to be something to initialize the settings with. Then only `django.conf.settings` would give the initialized settings.